weaver/wps_restapi/swagger_definitions.py

```diff
@@ -2406,7 +2421,13 @@ class ProcessListing(OneOfKeywordSchema):
 
 class ProcessCollection(ExtendedMappingSchema):
     processes = ProcessListing()
-    links = LinkList(missing=drop)
+
+
+class ProcessPagingQuery(ExtendedMappingSchema):
+    sort = ProcessSortEnum(missing=drop)
+    # if page is omitted but limit provided, use reasonable zero by default
+    page = ExtendedSchemaNode(Integer(allow_string=True), missing=0, default=0, validator=Range(min=0))
+    limit = ExtendedSchemaNode(Integer(allow_string=True), missing=None, default=None, validator=Range(min=0))
```

Comment:
min should be at least 1

Reply:
Good point. This will also avoid to explicitly check for the non zero in the code since it would fail validation otherwise.